package.json

```diff
@@ -49,7 +49,7 @@
     "react-dom-factories": "^1.0.0"
   },
   "peerDependencies": {
-    "react": "^0.14.9 || ^15.3.0 || ^16.0.0"
+    "react": "^0.14.9 || ^15.3.0 || ^16.0.0-beta"
```

Comment:
I'm torn on this. I don't think it's a big deal if you get peer warnings for betas.